.gitignore

```diff
@@ -0,0 +1,24 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
+/node_modules
+/.pnp
+.pnp.js
+
+# testing
+/coverage
+
+# production
+/build
+
+# misc
+.DS_Store
+.env.local
+.env.development.local
+.env.test.local
+.env.production.local
+package-lock.json
+
+npm-debug.log*
+yarn-debug.log*
```

Comment:
are we using npm or yarn ?

Reply:
npm yes, yarn i dont think so.

Reply:
okay please remove this and also resolve the merge conflict